packages/angular/package.json

```diff
@@ -31,8 +31,39 @@
     "./src/generators/move/move-impl": "./src/generators/move/move-impl.js",
     "./src/builders/*/schema.json": "./src/builders/*/schema.json",
     "./src/builders/*.impl": "./src/builders/*.impl.js",
+    "./src/builders/*": {
+      "import": "./src/builders/*.js",
+      "require": "./src/builders/*.js",
+      "types": "./src/builders/*.d.ts"
+    },
+    "./src/builders/*/*": {
```

Comment:
Is this too permissive?
This package already has exports defined, therefore files within sub folders in the builders directory would now be public api, restricting when and what changes we can make to them